solidity/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+artifacts/PetalLock.json
```

Comment:
This is a duplicate of the file referenced in the root `.gitignore`.  Recommend keeping it only in one of the two places.